src/Tizen.NUI/src/public/BaseComponents/ImageView.cs

```diff
@@ -260,12 +260,33 @@ private PropertyMap InternalImageMap
             {
                 if (_border == null)
                 {
-                    // Get current properties force.
+                    // Sync as current properties
                     // TODO: Need to make some flag that we only need cached property map.
+                    UpdateImage();
+
+                    // Get current properties force.
                     PropertyMap returnValue = new PropertyMap();
                     PropertyValue image = GetProperty(ImageView.Property.IMAGE);
                     image?.Get(returnValue);
                     image?.Dispose();
+
+                    // Update cached property map
+                    if(returnValue != null)
+                    {
+                        if(_imagePropertyMap != null)
+                        {
+                            _imagePropertyMap.Merge(returnValue);
```

Comment:
Merge is too expensive functionin dali + I think we don't need to merge here. just dispose and new is okay

Reply:
I think more detail yesterday but... Merge is neccessary job. (`_imagePropertyMap` 's role is actually "cache" . not "sync")
I'll prepare some more reasonable fixed patch next week.